tsconfig.json

```diff
@@ -4,9 +4,23 @@
         "noImplicitAny": true,
         "removeComments": true,
         "preserveConstEnums": true,
-        "lib": ["es6", "dom"]
+        "outDir": "./lib/",

```

Comment:
Does this mean that the output for the compiled js files is going to be the `./lib/` directory?

Reply:
Good catch, when you run "npm run compile", it will output the .js files into ```./lib/```.

Reply:
I meant, should the compiled js files go to `dist/`?

Reply:
we can. But I am thinking about getting rid of it since ```/dist``` already have a complied .js and .html that were generated by parcel. output files in ```./lib/``` seems a bit redundant at this point.

Reply:
<img width="649" alt="Screen Shot 2019-11-02 at 12 51 24 AM" src="https://user-images.githubusercontent.com/32313451/68067891-ed0c0900-fd0a-11e9-9016-a5ead10eba45.png">
<img width="817" alt="Screen Shot 2019-11-02 at 12 51 17 AM" src="https://user-images.githubusercontent.com/32313451/68067892-eda49f80-fd0a-11e9-821b-beead6ac020d.png">

Here are the two files, what do you think?

Reply:
Agreed. Maybe remove output to `lib/`.